src/components/sources/SourceHomeHeader.jsx

```diff
@@ -32,35 +32,33 @@ import ProcessingChip from '../common/ProcessingChip';
 import ConceptContainerDelete from '../common/ConceptContainerDelete';
 import CollapsibleDivider from '../common/CollapsibleDivider';
 
-const HIDDEN_ATTRIBUTES = {
-  canonical_url: 'url',
-  publisher: 'text',
-  purpose: 'text',
-  copyright: 'text',
-  content_type: 'text',
-  revision_date: 'date',
-  identifier: 'json',
-  contact: 'json',
-  jurisdiction: 'json',
-  meta: 'json',
-  collection_reference: 'text',
-  hierarchy_meaning: 'text',
-  experimental: 'boolean',
-  case_sensitive: 'boolean',
-  compositional: 'boolean',
-  version_needed: 'boolean',
-}
+const DEFAULT_VISIBLE_ATTRIBUTES = [
```

Comment:
Pick this from DEFAULT_CONFIG.
May be extract out defaultConfig.js for all orgs/sources/collections respectively and import.